maths/collision_between_rectangles.py

```diff
@@ -0,0 +1,27 @@
+rect1 = {"x": 10, "y": 10, "height": 30, "width": 50}
+
+rect2 = {"x": 20, "y": 30, "height": 40, "width": 30}
```

Comment:
These are not used within the actual code, so it should go within the `if __name__ == "__main__"` as test variables